### PR TITLE
speedup page loading

### LINKED ISSUE #117: Replace <router-link> with <NuxtLink>
Read more - https://nuxtjs.org/docs/features/nuxt-components/#the-nuxtlink-component

### PR DESCRIPTION
**Pull request by razbakov**
Closes #117

### CODE REVIEW

File: components/Card/TCardEvent.vue [hunk 1]
```diff
@@ -4,10 +4,10 @@
   </div>
   <div v-else class="flex space-x-4 border p-4">
     <div class="flex-grow">
-      <router-link
+      <NuxtLink
         :to="`/events/${node.id}`"
         class="font-bold leading-none underline hover:no-underline"
-        >{{ node.name }}</router-link
+        >{{ node.name }}</NuxtLink
```
Comment (best-lucky1030) [inline components/Card/TCardEvent.vue:10]: `>` is next line now.
I think it has in same line like: `</NuxtLink>`

**Review by best-lucky1030 — COMMENTED** [review]
All of `router-link` was replaced to `NuxtLink` ?

**Comment by razbakov** [comment]
> All of `router-link` was replaced to `NuxtLink` ?

Yes

File: components/Card/TCardEvent.vue (hunk 1 above)
Comment (razbakov) [reply components/Card/TCardEvent.vue:10]: It's formatted automatically by prettier.

**Pull request merged**